Tools/autotest/common.py

```diff
@@ -846,26 +846,46 @@ def disarm_motors_with_switch(self, switch_chan, timeout=20):
             if not self.mav.motors_armed():
                 self.progress("MOTORS DISARMED OK WITH SWITCH")
                 return True
-        self.progress("FAILED TO DISARM WITH SWITCH")
+        self.progress("Unable to DISARMED with switch")
         return False
 
-    def autodisarm_motors(self):
-        """Autodisarm motors."""
-        self.progress("Autodisarming motors")
-        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_GROUND_ROVER:  # NOT IMPLEMENTED ON ROVER
-            self.progress("MOTORS AUTODISARMED OK")
-            return True
+    def wait_autodisarm_motors(self):
+        """Wait for Autodisarm motors within disarm delay
+        this feature is only available in copter (DISARM_DELAY) and plane (LAND_DISARMDELAY)."""
+        self.progress("Wait autodisarming motors")
+        disarm_delay = None
+        if self.mav.mav_type in [mavutil.mavlink.MAV_TYPE_QUADROTOR,
```

Comment:
The DOCAHEXA is missing

Reply:
not supported by mav_type

Reply:
I think I have a PR for mavlink for that :)
@tridge can you merge it ?